client/scss/components/_header.scss

```diff
@@ -0,0 +1,289 @@
+//* @define header; weak
+
+.header {
+  padding-top: 0.8rem;
+}
+
+.header__upper-wrap {
+  border-bottom: 1px solid color('keyline-grey');
+}
+
+.header__upper,
+.header__lower {
+  display: flex;
+}
+
+.header__upper {
+  padding-bottom: 0.8rem;
+  justify-content: flex-end;
+  position: relative;
+
+  @include respond-to('medium') {
+    justify-content: space-between;
+  }
+}
+
+.header__lower {
+  justify-content: space-between;
+  position: relative;
+
+  &.is-active {
```

Comment:
Which makes me think that a modifier would be the best bet - again it's why BEM is here, to stop inheriting due to CSS's inability to namespace properly.